src/Utils/Value.php

```diff
@@ -256,12 +259,22 @@ private static function coercionError(
                 ? '.'
                 : '; ' . $subMessage);
 
+        $atPath = null;
+        if ($path !== null) {
+            $atPath = [];
+            do {
+                $key = $path['key'];
+                array_unshift($atPath, $key);
+                $path = $path['prev'];
+            } while ($path !== null);
+        }
+
         return new Error(
             $fullMessage,
             $blameNode,
             null,
             [],
-            null,
+            $atPath,
```

Comment:
This is not what `$path` is intended for, it is meant to represent the path of fields leading up to the error - not arguments! https://github.com/webonyx/graphql-php/blob/6f4e702c810f2902e43671701fb5a3e5a48bb68f/src/Error/Error.php#L40-L46